pkg/backends/cloudwatch/cloudwatch.go

```diff
@@ -26,11 +25,15 @@ const MAX_DIMENSIONS = 10
 // BackendName is the name of this backend.
 const BackendName = "cloudwatch"
 
+type ApiClient interface {
```

Comment:
nit: Do we want that to be more specific than ApiClient i.e. CloudwatchClient?

Reply:
yes we can

Reply:
My ide does give out with types starting with cloudwatch since it's the package name